GiniVision/Classes/Core/CameraViewController.swift

```diff
@@ -779,34 +821,44 @@ extension CameraViewController {
         }
     }
     
-    fileprivate func showPhotoLibraryPermissionDeniedError() {
-        let alertMessage = giniConfiguration.photoLibraryAccessDeniedMessageText
-        
-        let alertViewController = UIAlertController(title: nil, message: alertMessage, preferredStyle: .alert)
-        
-        alertViewController.addAction(UIAlertAction(title: "Abbrechen", style: .cancel, handler: { _ in
-            alertViewController.dismiss(animated: true, completion: nil)
-        }))
-        
-        alertViewController.addAction(UIAlertAction(title: "Zugriff erteilen", style: .default, handler: { _ in
-            alertViewController.dismiss(animated: true, completion: nil)
-            UIApplication.shared.openAppSettings()
-        }))
-        
-        self.present(alertViewController, animated: true, completion: nil)
-    }
-    
-    fileprivate func showNotValidDocumentError(message: String) {
-        
-        let alertViewController = UIAlertController(title: nil, message: message, preferredStyle: .alert)
-        alertViewController.addAction(UIAlertAction(title: "Abbrechen", style: .cancel, handler: { _ in
-            alertViewController.dismiss(animated: true, completion: nil)
-        }))
-        alertViewController.addAction(UIAlertAction(title: "Andere Datei wählen", style: .default, handler: { _ in
-            self.showImportFileSheet()
-        }))
+    func showErrorDialog(for error: Error, possitiveAction: (() -> Void)? = nil) {
+        let message: String
+        var cancelActionTitle: String = "Abbrechen"
```

Comment:
Not localized

Reply:
I didn't want to do this in this PR, but if it ok for you 🤷‍♂️